src/python/pants/backend/jvm/tasks/jar_publish.py

```diff
@@ -324,8 +324,6 @@ def register_options(cls, register):
                   'where many artifacts must align.')
     register('--transitive', default=True, type=bool,
              help='Publish the specified targets and all their internal dependencies transitively.')
-    register('--force', type=bool,
```

Comment:
Its not removed outright since `--force` is still readable from the global recursive scope - ie: command line stays the same and works.